cloud/index.rst

```diff
@@ -23,12 +23,125 @@ The key items you'll care about for each node:
 - Memory - The memory on each node varies based on the size of node you select
 - Cores - The cores on each node varies based on the size of node you select
 
-High Availability
-~~~~~~~~~~~~~~~~~
+Connecting to Citus Cloud
```

Comment:
The overview section doesn't seem like the right section for this content. Looking at the Menu bar on the left, I wouldn't expect information about connecting and roles in Overview.